style.css

```diff
@@ -386,7 +386,7 @@ h4{
   border: none;
   text-align: left;
   outline: none;
-  font-size: 15px;
+  font-size: 16px;
```

Comment:
Don't think it's because of font size, but the question text isn't centered.

![image](https://user-images.githubusercontent.com/54484616/133898039-187db573-9bb7-476b-8a65-c0dbbcbd16ee.png)